apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -340,6 +340,11 @@ type ClusterResourcePlacementStatus struct {
 	// +optional
 	SelectedResources []ResourceIdentifier `json:"selectedResources,omitempty"`
 
+	// ClusterResourceSnapshots is a list of clusterResourceSnapshot names.
+	// ClusterResourceSnapshot is used to store a snapshot of selected resources by a resource placement policy.
+	// A placement could produce multiple clusterResourceSnapshots to select large amount of resources.
+	ClusterResourceSnapshots []string `json:"clusterResourceSnapshots,omitempty"`
```

Comment:
Does the PlacementStatuses reflect the status of the listed ClusterResourceSnapshots or they can correspond to a previous version of the snapshots?

Basically, will we produce 2?

1. snapshot1, placementstatus for snapshot1
2. snapshot2, placementstatus for snapshot1
3. snapshot2, placementstatus for snapshot2